modules/core/arrow-instances-data/src/main/kotlin/arrow/instances/list.kt

```diff
@@ -0,0 +1,3 @@
+package arrow.instances
+
+object ListInstances
```

Comment:
Could you please put this file in arrow-data instead? That way we don't need to depend on the instances package when adding other typeclasses to List.

Reply:
Shouldn't these go to `arrow-core`?

Reply:
It's cheap so why not :D